content/geophysical_surveys/index.rst

```diff
@@ -15,23 +15,23 @@ Geophysical Surveys
 
 For each geophysical method, we will follow the framework:
 
-- Intro 
```

Comment:
Sounds good. I'll drop by your office later in this afternoon.

2016-02-17 12:59 GMT-08:00 Lindsey <notifications@github.com>:

> In content/geophysical_surveys/index.rst
> <https://github.com/ubcgif/em/pull/74#discussion_r53230440>:
>
> > @@ -15,23 +15,23 @@ Geophysical Surveys
> >
> >  For each geophysical method, we will follow the framework:
> >
> > -- Intro
>
> this is possibly the file where the merge conflict is happening. Do you
> want to try and resolve it? We can sit together if that would be helpful
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ubcgif/em/pull/74/files#r53230440>.
>